src/ui/MainMenu.cpp

```diff
@@ -64,13 +72,13 @@ namespace MainMenu {
 		{"Spell List", "B", hiddenBinding, emptyBinding},
 		{"Skill Sheet", "K", hiddenBinding, emptyBinding},
 		{"Autosort Inventory", "R", "ButtonLeftStick", emptyBinding},
-		{"Command NPC", "Q", "DpadY-", emptyBinding},
-		{"Show NPC Commands", "C", "DpadX+", emptyBinding},
-		{"Cycle NPCs", "E", "DpadX-", emptyBinding},
+		{"Command NPC", "Q", "DpadX+", emptyBinding},
```

Comment:
Maybe better to have <- (Command NPC) ^ (Open wheel) -> (Cycle) cause it's pretty and in order to follower menu glyph layout